docs/guides/_include/define-env.rst

```diff
@@ -1,26 +1,45 @@
-Define the environment
-----------------------
+Define disk variables 
+---------------------
 
 For convenience and to reduce the likelihood of errors, set environment variables that refer to the devices that
 will be configured during the setup. Verify your target disk devices with ``lsblk``.
```

Comment:
```suggestion
will be configured during the setup. Verify your target disk devices with ``lsblk``.

For many users, it is most convenient to place boot files (*i.e.*, ZFSBootMenu and any loader responsible for launching it) on the the same disk that will hold the ZFS pool. However, some users may wish to dedicate an entire disk to the ZFS pool and place these files on a separate device. A USB flash drive provides a convenient mechanism for booting whole-disk pools and is easily recreated if lost. Fortunately, this alternative configuration is easily realized by simply defining a few environment variables differently.
```

Reply:
I've added a variation of this text to the preamble of this section. Let me know what you think of it.